2024/political-donations-top-pacs_bubble/index.html

```diff
@@ -0,0 +1,43 @@
+<html lang="en">
+    <head>
+        <title>Top PACs By Donations</title>
+        <!-- PLACE ALL REPOSITORY CSS FILES IN A BRACKET LIKE THIS -->
+        <!-- build:css inline -->
+        <link rel="stylesheet" href="style.css" />
```

Comment:
Since most of the styles you're using are in the `main.css` file, you can import that and then add the `style.css` file with any additional CSS things in there.